de.metas.swat/de.metas.swat.base/src/main/java/de/metas/document/impl/DocumentPA.java

```diff
@@ -202,7 +201,7 @@ private int performUpdate(final int bPartnerLocationId, final String sql,
 		}
 	}
 
-	@Cached
+	@Cached(cacheName = Table_Name + "#by#" + COLUMNNAME_AD_Org_ID + "#" + COLUMNNAME_DocBaseType + "#" + COLUMNNAME_DocSubType + "#" + COLUMNNAME_IsDefault)
```

Comment:
The cache name is needed to make sure metasfresh know to reset this case, when the table name changes